usr/sbin/pkcshsm_mk_change/pkcshsm_mk_change.c

```diff
@@ -25,13 +26,19 @@
 #include <signal.h>
 #include <unistd.h>
 
-#include <pkcs11types.h>
+#include "pkcs11types.h"
 #include "p11util.h"
 #include "event_client.h"
 #include "pkcs_utils.h"
 #include "hsm_mk_change.h"
 #include "pin_prompt.h"
 
+#if defined(_AIX)
+#include <libgen.h>
+#define ELIBACC EINVAL
```

Comment:
do you need this here at all? You ifdef out all cases where ELIBACC is used....

Reply:
Yes - the cases that are `#ifdef`'d out are all situations where I'm interested in getting the actual `errno` that the function call returns. For cases where it just returns `ELIBACC` without getting an `errno` from the function call prior, it will now return `EINVAL`. See `pkcshsm-mk_change.c:1355` and the `if`-block below it, for instance.

Reply:
Ah OK, understood. Fine with me then.